packages/components/src/settings-form/types.ts

```diff
@@ -0,0 +1,32 @@
+export interface StringToString {
+	[ key: string ]: string;
+}
+
+export interface Field {
+	id: string;
+	type: 'text' | 'password' | 'checkbox' | 'select';
+	title: string;
+	label: string;
+	description?: string;
+	default?: string;
+	class?: string;
+	css?: string;
+	options?: StringToString;
+	tip?: string;
+	value?: string;
+	placeholder?: string;
+}
+
+export interface GetInputPropsReturn {
+	value: string;
+	checked: boolean;
+	selected: string;
+	onChange: ( name: string ) => void;
+	onBlur: () => void;
+	className: string | null;
```

Comment:
is this really string | null or should it be `className?`, either way `className` is defined on `React.HTMLAttributes`

here's the start of the React type's definition:

```
interface HTMLAttributes<T> extends AriaAttributes, DOMAttributes<T> {
        // React-specific Attributes
        defaultChecked?: boolean;
        defaultValue?: string | number | ReadonlyArray<string>;
        suppressContentEditableWarning?: boolean;
        suppressHydrationWarning?: boolean;

        // Standard HTML Attributes
        accessKey?: string;
        className?: string;
        contentEditable?: Booleanish | "inherit";
        contextMenu?: string;
        dir?: string;
        draggable?: Booleanish;
        hidden?: boolean;
        id?: string;
        lang?: string;
        placeholder?: string;
        slot?: string;
        spellCheck?: Booleanish;
        style?: CSSProperties;
        tabIndex?: number;
        title?: string;
        translate?: 'yes' | 'no';
        ...
```

Reply:
Yeah, looks like this was already addressed by the earlier refactor you suggested using the built in types.